cfgov/unprocessed/js/modules/util/breakpoint-state.js

```diff
@@ -22,40 +21,58 @@ function _inBreakpointRange( breakpointRange, width ) {
  * @returns {Object} An object literal with boolean
  *   isBpXS, isBpSM, isBpMED, isBpLG, isBpXL properties.
  */
-function get( width ) {
+function getBreakpoint( width ) {
   const breakpointState = {};
   let breakpointKey;
-  width = width || getViewportDimensions().width;
+  width = width || window.innerWidth;
 
   let rangeKey;
   // eslint-disable-next-line guard-for-in
-  for ( rangeKey in breakpointsConfig ) {
-    breakpointKey = 'is' + rangeKey.charAt( 0 ).toUpperCase() +
-                    rangeKey.slice( 1 );
-    breakpointState[breakpointKey] =
-      _inBreakpointRange( breakpointsConfig[rangeKey], width );
+  for ( rangeKey in varsBreakpoints ) {
+    breakpointState[rangeKey] = _inBreakpointRange(
+      varsBreakpoints[rangeKey],
+      width
+    );
   }
 
   return breakpointState;
 }
 
+// Constants for breakpoint groupings.
+const MOBILE = 'mobile';
+const TABLET = 'tablet';
+const DESKTOP = 'desktop';
+
 /**
- * Whether currently in the desktop view.
- * @returns {boolean} True if in the desktop view, otherwise false.
+ * Checks whether the current breakpoint is in a particular breakpoint group.
+ * @param {string} breakpointGroup - Breakpoint group names.
+ * @returns {boolean} True if in the breakpoint group, otherwise false.
  */
-function isInDesktop() {
+function isIn( breakpointGroup ) {
```

Comment:
Harping on semantics again but `isIn` isn't super descriptive and because we're not passing any arguments to the private function `getBreakpoint()` below, this function only tests the current viewport's width. So I'd vote for a name like `viewportIsIn` or `isInBreakpointRange`.

Reply:
Updated to `viewportIsIn`